designs/virtualization/4.3/vm-details/dedicated-resources/dedicated-resources.md

```diff
@@ -0,0 +1,26 @@
+---
+parent: Virtualization
+version: 4.3
```

Comment:
I think this would be for version 4.4 right (OCP)?

Reply:
Changed to 4.4.
I also added:
"CNV-version 2.3"
@andybraren is that legit?